providers/redfish/firmware.go

```diff
@@ -120,59 +124,82 @@ func (c *Conn) FirmwareInstall(ctx context.Context, component, applyAt string, f
 
 	// The response contains a location header pointing to the task URI
```

Comment:
lets move this taskID parsing section into a method called - `TaskIDFromLocationURI()` ?

Reply:
Done